### PR TITLE
app/vmctl: Allow vmnative exports to skip HTTP keepalive.

### PR DESCRIPTION
**Pull request by baconmania**
For VictoriaMetrics deployments in cloud environments, containerized components move around across various virtual machine instances. When this happens in the middle of a longer `vmctl` export run, we see `vmctl` quit with the error `use of closed network connection`. This PR adds a flag to allow `vmctl` to use non-persistent HTTP connections to avoid this problem.

### CODE REVIEW

File: app/vmctl/flags.go [hunk 1]
```diff
@@ -325,6 +325,8 @@ const (
 	vmNativeFilterTimeEnd   = "vm-native-filter-time-end"
 	vmNativeStepInterval    = "vm-native-step-interval"
 
+	vmNativeNoHttpKeepAlive = "vm-native-no-http-keep-alive"
```
Comment (dmitryk-dk) [inline app/vmctl/flags.go:328]: It would be better to rename this flag.
```golang
vmNativeDisableHttpKeepAlive = "vm-native-disable-http-keep-alive"
```

Comment (baconmania) [reply app/vmctl/flags.go]: Done!

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Review by hagen1778 — APPROVED** [review]
LGTM!

**Comment by hagen1778** [comment]
@dmitryk-dk would you mind adding a note to changelog and maybe update vmctl docs if that's appropriate?

**Comment by valyala** [comment]
FYI, this pull request has been included in [v1.89.0](https://docs.victoriametrics.com/CHANGELOG.html).

**Pull request merged**